pyvista/examples/examples.py

```diff
@@ -412,3 +412,65 @@ def load_nut():
 
     """
     return pyvista.read(os.path.join(dir_path, 'nut.ply'))
+
+
+def _cart2sphe(x, y, z):
+    """Convert 3D Cartesian coordinates to spherical coordinates.
+
+    Parameters
+    ----------
+    x, y, z : np.ndarray
+        Coordinates.
+
+    """
+    xy2 = x**2 + y**2
+    r = np.sqrt(xy2 + z**2)
+    theta = np.arctan2(np.sqrt(xy2), z)  # the polar angle in radian angles
+    phi = np.arctan2(y, x)  # the azimuth angle in radian angles
+    phi[phi < 0] += np.pi * 2  # np.arctan2 returns the angle in the range [-pi, pi]
+
+    return r, theta, phi
+
+
+def load_hydrogen_orbital(n=1, l=0, m=0, norm=True):
+    """Load the hydrogen wave function for a :class:`pyvista.UniformGrid`.
+
+    Ispired by `Hydrogen Wave Function
+    <http://staff.ustc.edu.cn/~zqj/posts/Hydrogen-Wavefunction/>`_
```

Comment:
```suggestion
    <http://staff.ustc.edu.cn/~zqj/posts/Hydrogen-Wavefunction/>`_.
    
    Parameters
    ----------
    n : int, default: 1
        Principal quantum number. Must be a positive integer.
    
    l : int, default: 0
        Azimuthal quantum number. Must be a non-negative integer strictly smaller than ``n``.
    
    m : int, default: 0
        Magnetic quantum number. Must be an integer ranging from ``-l`` to ``l`` (inclusive).
```
(I haven't checked line lengths, feel free to edit.)